_studio/shared/umc/codec/h264_dec/src/umc_h264_mfx_supplier.cpp

```diff
@@ -388,7 +388,7 @@ Status MFXTaskSupplier::DecodeSEI(NalUnit *nalUnit)
         bitStream.Reset((uint8_t*)swappedMem.GetPointer(), (uint32_t)swappedMem.GetDataSize());
 
         NAL_Unit_Type nal_unit_type;
-        uint32_t nal_ref_idc;
+        uint32_t nal_ref_idc = 0;
```

Comment:
Why we don't initialize nal_unit_type then? And what we actually get in the bitStream.GetNALUnitType(nal_unit_type, nal_ref_idc); - both values or only one?

Reply:
I would either initialize both or have error check for GetNALUnitType.

Reply:
Done